proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -43,7 +44,7 @@ message MsgCreateValidator {
   ];
   string                   delegator_address = 4 [(gogoproto.moretags) = "yaml:\"delegator_address\""];
   string                   validator_address = 5 [(gogoproto.moretags) = "yaml:\"validator_address\""];
-  string                   pubkey            = 6;
+  google.protobuf.Any      pubkey            = 6;
```

Comment:
Inside staking.proto, Validator.ConsensusPubKey also needs to be Any

Reply:
Curious @robert-zaremba if you had intended to refactor `Validator.ConsensusPubKey` in this PR, or to tackle it later in the subsequent one?

Reply:
It's part of this PR. All helper functions are made to hide the decoding and message creation validation in tests.

Reply:
Subsequent one.